backend/app/api/endpoints/internal/conversion_callback.py

```
@@ -0,0 +1,138 @@
+        converted_name=request.converted_name,
+        converted_file_size=request.file_size,
+    )
+    if not succeeded:
+        return ConversionCompletedResponse(
+            ok=True, skipped=True, skip_reason="stale_conversion"
+        )
+
+    # Step 2: Overwrite attachment with markdown
+    try:
+        markdown_bytes = base64.b64decode(request.markdown_bytes, validate=True)
+    except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)